core/authors/views.py

```diff
@@ -90,7 +90,22 @@ def get_external_profile(self, request):
         if not success:
             return Response("Failed to connect with external server: "+server.get_base_url(), status=500)
         # for some reason we throw friends in here
-        profile["friends"] = []
+        if "friends" not in profile:
+            profile['friends'] = []
+            server2 = ServerUtil(url=authorUrl.split('/author')[0])
+            if server2.is_valid():
+                success, friends = server2.get_author_friends(authorUrl.split("author/")[1])
+                if not success:
```

Comment:
Technically setting friends to an empty array here isn't needed since it's done on line 94